packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -372,6 +361,11 @@ export function TransferPanel() {
       throw 'Signer is undefined'
     }
 
+    if (destinationAddressError) {
+      console.error(destinationAddressError)
+      return
+    }
```

Comment:
we don't need to show it on console?
is this intended?

Reply:
imo it's good as it narrows down the potential issue so we can debug it easier, especially if it's a user facing issue

Reply:
maybe fire a Sentry event?

Reply:
This case is one of them 'it should never get here' things. Our UI is already guarding against this happening. So I don't think we will ever get issues with that to justify sending a Sentry event here. And if it happens for any reason, reloading the page should fix it.

We do similar checks just above this.